sitemedia/js/components/RangeFacet.js

```diff
@@ -0,0 +1,38 @@
+import { mapActions } from 'vuex'
+
+export default Vue.component('RangeFacet', {
+    template: `
+    <div class="range-facet">
+        <div class="inputs">
+            <sui-input :placeholder="minVal" />
+            <label>to</label>
+            <sui-input :placeholder="maxVal" />
+        </div>
+        <div class="histogram">
+        </div>
+    </div>
+    `,
+    props: {
+        label: String,
+        name: String,
+        width: Number,
+        choices: Array
+    },
+    computed: {
+        minVal() {
+            return this.choices
+                .map(choice => parseInt(choice.value))
+                .reduce((acc, cur) => cur < acc ? cur : acc, Infinity)
+        },
+        maxVal() {
+            return this.choices
+                .map(choice => parseInt(choice.value))
+                .reduce((acc, cur) => cur > acc ? cur : acc, -Infinity)
+        }
+    },
+    methods: {
+        ...mapActions([
+            'setRangeFacet',
```

Comment:
I am probably missing something here, but I see getters for `setRangeFacetMin` and `setRangeFacetMax` but not `setRangeFacet`.

Reply:
Yeah, these are placeholders - I started out using a similar approach to the normal facets (one `set` function) but realized I needed two and haven't changed it yet. Coming soon...